packages/replay/src/constants.ts

```diff
@@ -45,3 +45,8 @@ export const SLOW_CLICK_SCROLL_TIMEOUT = 300;
 
 /** When encountering a total segment size exceeding this size, stop the replay (as we cannot properly ingest it). */
 export const REPLAY_MAX_EVENT_BUFFER_SIZE = 20_000_000; // ~20MB
+
+/** Replays must be min. 4s long before we send them. */
+export const MIN_REPLAY_DURATION = 4_000;
```

Comment:
Let's make this 5

Reply:
Maybe let's make it `4_999`, wanted to ensure we avoid race conditions with the 5s flush delay...?